src/app/components/image-card/image-card.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="card p-1 m-1">
+
+    <app-image class="m-auto" [image]=image></app-image>
+
+    <div *ngIf="!isOnlyDisplaying">
+        <div class="d-flex justify-content-between pt-1 mt-1 border-top border-secondary">
+            <button type="button" class="btn btn-outline-dark btn-sm py-0"><i class="bi bi-eye"></i></button>
```

Comment:
Please move <button> block content (I mean <i class ... /i>) to the next line.
And fix same cases in file.